app/src/androidTest/java/com/github/campus_capture/bootcamp/MapVoteTest.java

```diff
@@ -39,6 +39,7 @@
 import org.hamcrest.Matcher;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Ignore;
```

Comment:
I think this is an old import, if you can just optimize the imports it would be perfect.